crates/starknet-devnet/tests/test_advancing_time.rs

```diff
@@ -259,18 +257,33 @@ mod advancing_time_tests {
         // wait 1 second
         tokio::time::sleep(time::Duration::from_secs(1)).await;
 
-        // check if after mint timestamp > last block, check if inside buffer limit
-        devnet.mint(DUMMY_ADDRESS, DUMMY_AMOUNT).await;
-        let mint_block = devnet.get_latest_block_with_tx_hashes().await.unwrap();
-        assert_gt_with_buffer(mint_block.timestamp, empty_block.timestamp);
+        // check if after create block timestamp > last block, check if inside buffer limit
+        devnet.create_block().await.unwrap();
+        let latest_block = devnet.get_latest_block_with_tx_hashes().await.unwrap();
+        assert_gt_with_buffer(latest_block.timestamp, empty_block.timestamp);
```

Comment:
So far minting was used just to induce the creation of a new block (with new timestamp)?

Reply:
Yes this is why I changed it to create_block()